table/block_based_table_builder.cc

```diff
@@ -651,11 +651,15 @@ void BlockBasedTableBuilder::WriteBlock(const Slice& raw_block_contents,
   auto type = r->compression_type;
   Slice block_contents;
   bool abort_compression = false;
+  
+  StopWatchNano timer(Env::Default(), true);
```

Comment:
Are you sure that default env is the one that needs to be used? This defeats the custom env that is specified in the options.